tools/distrib/python/grpcio_tools/grpc/tools/command.py

```diff
@@ -52,19 +71,4 @@ def run(self):
     # directory is provided as an 'include' directory. We assume it's the '' key
     # to `self.distribution.package_dir` (and get a key error if it's not
     # there).
-    proto_files = []
-    inclusion_root = os.path.abspath(self.distribution.package_dir[''])
-    for root, _, files in os.walk(inclusion_root):
-      for filename in files:
-        if filename.endswith('.proto'):
-          proto_files.append(os.path.abspath(os.path.join(root, filename)))
-
-    for proto_file in proto_files:
-      command = [
-          'grpc.tools.protoc',
-          '--proto_path={}'.format(inclusion_root),
-          '--python_out={}'.format(inclusion_root),
-          '--grpc_python_out={}'.format(inclusion_root),
-      ] + [proto_file]
-      if protoc.main(command) != 0:
-        sys.stderr.write('warning: {} failed'.format(command))
+    build_package_protos(self.distribution.package_dir[''])
```

Comment:
Someday there'll be an option to configure a different package directory, or maybe a way to just walk all package directories without tripping on anything unforeseen. Someday.